spec/like_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  subject do
+    user = User.new(name: 'Tom', photo: 'gama.png', bio: 'I am a decent person', posts_counter: 0)
+    Post.new(title: 'Wow', text: 'Hello moto',
+             likes_counter: 0, comments_counter: 0, user_id: user.id)
+  end
+
+  user = User.new(name: 'Tom', photo: 'gama.png', bio: 'I am a decent person', posts_counter: 0)
+
+  before { user.save }
+
+  it 'should return most recent comments' do
+    user.likes.create!(post: subject)
+    user.likes.create!(post: subject)
+
+    expect(subject.likes_counter).to eql 2
+  end
```

Comment:
- [ ] Your test is failing. Kindly fix that.

  ![image](https://user-images.githubusercontent.com/36542986/221371934-c0c3cd74-b06d-40fb-bb0c-a0236fd8bf0c.png)

- [ ] **[OPTIONAL]** I think it would be a better idea to move this test into a `spec/models` folder, instead of just leaving it as `spec` as it makes for a better organization.